cmd/oras/cmd/example_test.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd_test
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"oras.land/oras/cmd/oras/cmd"
+)
+
+// Example gives playable snippets of using oras CLI in command-line scripting
+// via Golang.
+func Example() {
+	// change below array to play with your own cmd args:
+	args := []string{"repo", "ls", "mcr.microsoft.com"}
```

Comment:
Even though this is a testable example, but it don't have any `// output` comment so it will be ignored by unit test, i.e. it will be a runnable example in godoc after the next release but it won't be run in CI.

Reply:
Looks like checking in the file will heavily degrade the test coverage ratio by 34%, need to properly set codecov to ignore command type code.